modules/tinymce/src/plugins/lists/main/ts/api/Commands.ts

```diff
@@ -43,8 +51,10 @@ const register = (editor) => {
     flattenListSelection(editor);
   });
 
-  editor.addCommand('mceListProps', () => {
-    Dialog.open(editor);
+  registerDialog(editor);
+
+  editor.addCommand('SetListAttributes', (ui, detail) => {
```

Comment:
This should probably be called `mceSetListAttributes` to keep with the current naming convention (ie custom commands are prefixed with `mce`).

- [ ] Change command name to `mceSetListAttributes`

Reply:
I'm also wondering if this maybe should be called `mceListMutation` (or similar) to go with the event of a similar name? If we do that, we probably need to change the data type to something like `{ attrs: Record<string, string> }`. That would then make this much more flexible for future potential, if say we needed to set styles or classes, etc...

Reply:
I have no attachment to the command name, I'm happy to go with whatever the team wants. I am also concerned about the scalability of this to later set CSS attributes; `ApplyUnorderedListStyle` and `ApplyOrderedListStyle` from `advlist` could be replaced with this command if we do that.

A sub-object seemed like overkill but is probably the best approach.

Reply:
We don't seem to use `Mutate` in our event names anywhere, how about `mceUpdateList`?

Reply:
The event name is `ListMutation`, but it's used when changing the types mostly. That suggestion seems fine to me, though we do normally try to keep the list name first, eg `mceListUpdate`.

Reply:
ah I was only looking at `editor.addCommand` calls, not `editor.fire` as well. oops.

list update goes well with list props. I'll use that.